templates/editorial_queue/queue.html

```diff
@@ -0,0 +1,6 @@
+{% extends "baselayout.html" %}
+{% load i18n %}
+{% block bootstrap3_title %}{% blocktrans %}Glosses in the Queue{% endblocktrans %} | {% endblock %}
```

Comment:
```suggestion
{% block bootstrap3_title %}{% blocktrans %}Editorial queue{% endblocktrans %} | {% endblock %}
```

Maybe use the title to match the feature?

Reply:
Thanks @joshmcarthur , changed the text :slightly_smiling_face: